kuzzleio/sdk-csharp
# Csharp from go

## What does this PR do ?

Wrap the Csharp SDK from the go SDK with swig

### How should this be manually tested?

You should have `mono-mcs` installed

```sh
git clone --recursive git@github.com:kuzzleio/sdk-csharp.git
git submodule update --init --recursive
cd sdk-csharp
make
```
